11-foundations-randomization.qmd

```diff
@@ -716,46 +716,47 @@ The test statistic in the opportunity cost study was the difference in the propo
 In each of these examples, the **point estimate** of the difference in proportions was used as the test statistic.
 :::
 
-When the p-value is small, i.e., less than a previously set threshold, we say the results are **statistically significant**\index{statistically significant}.
-This means the data provide such strong evidence against $H_0$ that we reject the null hypothesis in favor of the alternative hypothesis.
+When the p-value is small, i.e., less than a previously set threshold, we say the results are **statistically discernible**\index{statistically significant}\index{statistically discernible}.
+This means the data provide such strong evidence against $H_0$ that we reject the null hypothesis in favor of the alternative hypothesis.^[Many texts use the phrase "statistically significant" instead of "statistically discernible".  We have chosen to use "discernible" to indicate that a precise statistical event has happened, as opposed to an interesting effect which may or may not fit the statistical definition of discernible or significant.]
```

Comment:
```suggestion
This means the data provide such strong evidence against $H_0$ that we reject the null hypothesis in favor of the alternative hypothesis.^[Many texts use the phrase "statistically significant" instead of "statistically discernible".  We have chosen to use "discernible" to indicate that a precise statistical event has happened, as opposed to a notable effect which may or may not fit the statistical definition of discernible or significant.]
```